### ISSUE TITLE
Different shaped input and target

### ISSUE THREAD

**Issue by paul22** [jsrimr/udacity-final-project#1]
Hi

Your projects looks interesting and so I've attempted to run your baseline, but I get an error after episode 28.
-- # of episode: 28, avg score: -0.8999, asset=1001.00,
Warning (from warnings module):
  File "C:\Users\Paul\OneDrive\Documents\udacity-final-project-master\DQNTradingAgent\dqn_agent.py", line 182
    loss = F.smooth_l1_loss(q_eval, q_target.detach(), reduction='none')
UserWarning: Using a target size (torch.Size([64, 1, 32])) that is different to the input size (torch.Size([64, 32, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.

I've made two simple changes to train_leverage.py. I commented out the following so that I didn't have to install neptune:-
-- import neptune.new as neptune
-- from config import API_TOKEN

I'm still waiting for the results, but from my limited experience the above warning doesn't sound great.
Thanks

Hi after the episode 29 data is printed the script seems to stop and not produce further output :-(
